doc/cephadm/host-management.rst

```diff
@@ -228,6 +228,83 @@ create a new CRUSH host located in the specified hierarchy.
 
 See also :ref:`crush_map_default_types`.
 
+OS Tuning Profiles
+==================
+
+Cephadm supports managing os tuning profiles that allow users to apply a set of
+sysctl settings to a given set of hosts. To do this, first create a yaml spec
+of the following format
+
+.. code-block:: yaml
+
+    profile_name: 23-mon-host-profile
+    placement:
+      hosts:
+        - mon-host-01
+        - mon-host-02
+    settings:
+      fs.file-max: 1000000
+      vm.swappiness: '13'
+
+Then apply the tuned profile with::
+
+    ceph orch tuned-profile apply -i <tuned-profile-file-name>
+
+This profile will then be written to /etc/sysctl.d/ on each host matching the
+given placement and `sysctl --system` will be run on the host.
```

Comment:
Suggest: This profile file will then be written to  \`\`/etc/sysctl.d\`\`

We're a bit inconsistent about what gets monospaced, I think this should be.